lib/registrant.rb

```diff
@@ -0,0 +1,30 @@
+class Registrant 
+    attr_reader :age,
+                :name
+
+    attr_accessor :license_data
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = {written: false, 
+                        license: false, 
+                        renewed: false}
+    end
+
+    def permit?
+        @permit = @permit
+    end
+
+    def earn_permit
+        if @permit == false
+            return @permit = true
+        else
+            @permit
+        end
+
+    end
+end
```

Comment:
Your indentation looks really large in this file. When you're working in VSCode, the bottom right of the screen shows how many spaces are being added for an indent. Typically, you want 2 spaces